soul-spring-boot-starter/soul-spring-boot-starter-client/soul-spring-boot-starter-client-sofa/pom.xml

```diff
@@ -29,6 +29,11 @@
     <artifactId>soul-spring-boot-starter-client-sofa</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>com.alipay.sofa</groupId>
+            <artifactId>runtime-sofa-boot-starter</artifactId>
+            <version>3.1.4</version>
+        </dependency>
```

Comment:
why add this

Reply:
In soul-client-sofa, this dependence is provided, either add this or hit NoClassDefFoundError. And I think if users add the soul-spring-boot-starter-client-sofa that means they need a sofa starter

Reply:
If someone need sofa starter , he should import it himself.